apps/ledger-live-desktop/src/renderer/live-common-setup.ts

```diff
@@ -17,6 +17,9 @@ setEnvOnAllThreads("USER_ID", getUserId());
 // This defines our IPC Transport that will proxy to an internal process (we shouldn't use node-hid on renderer)
 registerTransportModule({
   id: "ipc",
-  open: id => retry(() => IPCTransport.open(id)),
+  open: (id: string) => {
+    if (id !== "ipc") return;
```

Comment:
Can you add a comment explaining why the id could be something other than `ipc` (but also `vault-transport`) pls ?